databricks/koalas/frame.py

```diff
@@ -400,6 +403,40 @@ def groupby(self, by):
         gp = self._sdf.groupby(by)
         return PandasLikeGroupBy(self, gp, None)
 
+    @derived_from(pd.DataFrame)
+    def isin(self, values):
+        """
+        Check whether values are contained in Series.
+
+        :param values: iterable or dict
+                       The sequence of values to test. If values is a dict,
+                       the keys must be the column names, which must match.
+                       Series and DataFrame are not supported.
+        :return: :class:`DataFrame`
+        """
```

Comment:
and doc test + inlining with Pandas doc (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.isin.html#pandas.DataFrame.isin)